lips/lip-1.mdx

```diff
@@ -510,12 +531,14 @@ The `receiver_metadata_signature` will then be placed into the `recipient_signat
 Once both sender and receiver have a status of 'ready_for_settlement', the transaction may then be submitted on-chain by the sender VASP. This submission will utilize the `recipient_signature` which was provided by the receiver VASP (generated via [Recipient Signature](#recipient-signature).  The sender VASP will now generate a transaction via the following and then submit the transaction on-chain:
 
 ```
-metadata = TravelRuleMetadatav0 {
-  off_chain_reference_id: "123abc", /* The off-chain reference ID specified by `reference_id` in the Off-chain APIs */
-};
+metadata = Metadata::TravelRuleMetadata(
+  TravelRuleMetadata::TravelRuleMetadataVersion0(
+    TravelRuleMetadatav0 {
+      off_chain_reference_id: "123abc", /* The off-chain reference ID specified by `reference_id` in the Off-chain APIs */
+}));
+
 
-lcs_metadata = lcs(MetadataType::TravelRuleMetadataType(
-        TravelRuleMetadata::TravelRuleMetadataVersion0(metadata))),
+lcs_metadata = lcs.serialize(metadata, Metadata);
```

Comment:
Since we haven't actually imported anything for this example, it's confusing what "Metadata" is.  Can we do the imports or use the full name? It makes sense in LIP 4 because the reader has the context, but not as much here

Reply:
Sounds good (Metadata is the old MetadataType), let me prefix with libra_types then